cocos/platform/CCPosixFileStream.cpp

```diff
@@ -178,7 +178,7 @@ int PosixFileStream::close()
     return internalClose();
 }
 
-long PosixFileStream::seek(int64_t offset, int origin)
+int PosixFileStream::seek(int64_t offset, int origin)
 {
     const auto result = _iof->seek(_handle, static_cast<int32_t>(offset), origin); // this returns -1 for error, and resulting offset on success
```

Comment:
Needs explicit int64_t offset, and there are lseeki64 for windows and unix is int64_t by default

Reply:
Ok, I'll change it soon

Reply:
You can add
#if defined(_WIN32)
// ...
#  define posix_lseek64 ::_lseeki64
# else
#if defined(__APPLE__)
#  define posix_lseek64 ::lseek
#else
#  define posix_lseek64 ::lseek64
#endif
#endif

Reply:
Where should that section be added?

Reply:
https://github.com/adxeproject/adxe/blob/dev/cocos/platform/CCPosixFileStream.h

Reply:
All good, I'll sort it out soon